src/pykeen/models/unimodal/ntn.py

```diff
@@ -71,180 +73,32 @@ def __init__(
         :param num_slices: The number of slices in the parameters
         :param non_linearity: A non-linear activation function. Defaults to the hyperbolic
             tangent :class:`torch.nn.Tanh`.
+        :param non_linearity_kwargs: If the ``non_linearity`` is passed as a class, these keyword arguments
+            are used during its instantiation.
         :param entity_initializer: Entity initializer function. Defaults to :func:`torch.nn.init.uniform_`
         :param kwargs:
             Remaining keyword arguments to forward to :class:`pykeen.models.EntityEmbeddingModel`
         """
         super().__init__(
+            interaction=NTNInteraction(
+                activation=non_linearity,
+                activation_kwargs=non_linearity_kwargs,
+            ),
             entity_representations=EmbeddingSpecification(
                 embedding_dim=embedding_dim,
                 initializer=entity_initializer,
             ),
+            relation_representations=[
+                # w: (k, d, d)
+                EmbeddingSpecification(shape=(num_slices, embedding_dim, embedding_dim)),
+                # vh: (k, d)
+                EmbeddingSpecification(shape=(num_slices, embedding_dim)),
+                # vt: (k, d)
+                EmbeddingSpecification(shape=(num_slices, embedding_dim)),
```

Comment:
This might already been documented somewhere, but we split the original `V_r` matrix into two parts, to separate `V_r [h; r] = V_r^h h + V_r^t t`. The latter is more efficient, if `h` and `t` are not of the same shape, e.g., since we are in a `score_h` / `score_t` setting..

Reply:
Done in 66cf699b